<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-LPT61RQN" failures="1" tests="5" name="Default test" time="21.04" errors="0" timestamp="2021-06-08T00:01:54 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.417"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.201">
    <failure type="junit.framework.AssertionFailedError" message="Test fail">
      <![CDATA[junit.framework.AssertionFailedError: Test fail
at junit.framework.Assert.fail(Assert.java:57)
at stepDef.steps.user_should_able_to_export_the_report(steps.java:77)
at âœ½.User should able to export the report(file:///C:/Users/anjan/eclipse-workspace/CucumberProject/src/test/java/features/feature2.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.308"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.2"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.254"/>
</testsuite> <!-- Default test -->
